plugins/lookup/revbitspss.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, RevBits <info@revbits.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+name: revbitspss
+author: RevBits (@info) <info@revbits.com>
+short_description: Get secrets from RevBits PAM server
+version_added: 1.0.1
+description:
+    - Uses the revbits-ansible Python SDK to get Secrets from RevBits PAM
+      Server using API key authentication with the REST API.
+requirements:
+    - revbits-ansible - https://pypi.org/project/revbits-ansible/
+options:
+    _terms:
+        description:
+            - This will be an array. First index will conatin another array of keys for secrets which you want to fetch from RevBits PAM
```

Comment:
```suggestion
            - This will be an array. First index will conatin another array of keys for secrets which you want to fetch from RevBits PAM.
```
All paragraphs should have complete sentences. (Same below.)